unlock-app/src/components/content/event/EventDetails.tsx

```diff
@@ -221,30 +223,47 @@ export const EventDetails = ({
         ]}
       />
       <div className="flex flex-col gap-4">
-        {isOrganizer && (
-          <div className="flex flex-row-reverse gap-2 ">
-            <Button
-              onClick={() => {
-                router.push(`/event/${eventProp.slug}/settings`)
-              }}
-            >
-              <div className="flex items-center gap-2">
-                <Icon icon={TbSettings} size={20} />
-                <span>Settings</span>
-              </div>
-            </Button>
-            <Button
-              onClick={() => {
-                router.push(`/event/${eventProp.slug}/attendees`)
-              }}
-            >
-              <div className="flex items-center gap-2">
-                <Icon icon={FaUsers} size={20} />
-                <span>Attendees</span>
-              </div>
-            </Button>
-          </div>
-        )}
+        <div className="flex flex-col-reverse px-4 md:px-0 md:flex-row-reverse gap-2 ">
+          {isOrganizer && (
```

Comment:
Can you check that this still works for someone who is NOT a lock manager but has been added as a verifier? They should not be able to see attendees or settings...